R/pkg/R/functions.R

```diff
@@ -2047,17 +2049,15 @@ setMethod("countDistinct",
 #' \code{concat}: Concatenates multiple input columns together into a single column.
 #' If all inputs are binary, concat returns an output as binary. Otherwise, it returns as string.
 #'
-#' @rdname column_string_functions
+#' @rdname column_collection_functions
 #' @aliases concat concat,Column-method
 #' @examples
 #'
 #' \dontrun{
 #' # concatenate strings
 #' tmp <- mutate(df, s1 = concat(df$Class, df$Sex),
 #'                   s2 = concat(df$Class, df$Sex, df$Age),
```

Comment:
here `df` is referencing https://github.com/huaxingao/spark/blob/b7b50b6ad11569c8e2c77f300a2c8cdd7064c959/R/pkg/R/functions.R#L141

since rdname is changed, df no longer work here. since you have added an example (perhaps add more? or just add a simple one like the ones here) https://github.com/apache/spark/pull/21255/files#diff-d97f9adc2dcac0703568c799ff106987R222 let's remove the whole example block L 2054-2061 here, if we add one like `concat(df$Class, df$Sex, df$Age, df$Class)` (change to valid column names) to after L 222?

Reply:
for background, we want to keep example runnable as much as possible..